### ISSUE TITLE
[BE] Establishing the connection between EmailProxy and TalktixChatAPI

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#40]
Expansion to #38

- define concrete server ports 

## Acceptance criteria
- [x] messages from EmailProxy is succesfully received by TalktixChatAPI
- [x] messages from TalktixChatAPi are successfully received by EmailProxy 
- [x] tests have been conducted

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
